other/field-types/joinByArray.md

```diff
@@ -0,0 +1,109 @@
+
+### Example 2
+
+```javascript
+{
+  name: '_departments',
+  label: 'Departments',
+  // We could skip this since it is the same as the name of the join,
+  // plus an s. Since we specified it, it must match the `name` property
+  // of another type (it'll be singular)
+  withType: 'department',
+  type: 'joinByArray',
+  filters: {
+    // Fetch just enough information
+    projection: {
+      title: 1,
+      slug: 1,
+      type: 1,
+      tags: 1
+    }
+  },
+  relationship: [
+    {
+      name: 'jobTitle',
+      label: 'Job Title',
+      type: 'string'
+    }
+  ]
+}
+```
+
+Since there is a relationship, the `_departments` property will be an array of objects with `item` and `relationship` properties. The `item` property will be the actual department, and the `relationship` property will contain the relationship fields, which are unique to this person.
+
+###Inline relationship fields
+
+Sometimes, expecting users to click a special button to access a separate modal dialog box to edit relationship fields isn't worth it. Users just don't find it, or the fields are few enough that it would make more sense to add the form field directly to the chooser.
+
+You can do this with the `inline: true` flag \(since 2.6.0\):
+
+### Example 3
+
+```javascript
+relationship: [
+  {
+    name: 'jobTitle',
+    label: 'Job Title',
+    type: 'string',
+    inline: true
+  }
+],
+```
+
+If you have a mix of inline and regular fields, you'll still get the option of opening the modal, but for data integrity reasons fields are presented only in one place or the other.
+
+## Settings
+
+|  Property | Type   | Default | Description | 
+|---|---|---|---|
+| name | `string` | | Sets the name of the field to join with (must begin with `_`) |
+| withType | `string` | | The name of the related type, if it differs from the name of the join. If you do not set `withType`, then the name of the join must match the name of the related type, with a leading `_` added.  |
+| idsField | `string` | | Sets the name of the property in which to store the id. The id is set automatically otherwise. |
+| ifOnlyOne | `boolean` | false | If true, it will not carry out the join if you are working with more than one document |
+| withJoins | `array` |  | If you need to carry out nested joins, set to an array containing those join field names. You may also use "dot notation" in these names to indicate that you want to follow a series of joins between related types.
+| label | `string` | | Sets the label of the field that the user sees |
+| filters | `object` | | Provide a list of cursor filters to limit acceptable options for the join |
+| inline | `boolean` | false | If true, adds the form field directly to the chooser |
```

Comment:
Incorrect, `inline` is a sub-property of each entry in `relationship`, which should be in this table but currently is not (see above for an example that describes it).

Reply:
Okay. Fixed.

Reply:
I think the relationship property is simple enough to define without a separate article.